src/docs/truffle/reference/configuration.md

```diff
@@ -131,6 +131,24 @@ For each network, you can specify `host` / `port`, `url`, or `provider`, but not
 - https://sandbox.truffleteams.com/yoursandboxid
 - wss://sandbox.truffleteams.com/yoursandboxid
 
+#### Special networks: `test` and `develop`
```

Comment:
Can we introduce the term for these as "managed networks"? (Or a better name if there is one)

```suggestion
#### Special "managed" networks: `test` and `develop`
```

Reply:
Sure, I'll go introduce that term a bit more thoroughly.

Reply:
OK, I updated it to use that term in a few places. :)  (I also reordered some things while I was at it.)